src/core/agent/services/signifyNotificationService.ts

```diff
@@ -367,6 +378,32 @@ class SignifyNotificationService extends AgentService {
         await this.markNotification(notif.i);
         return;
       }
+      const notifications = await this.notificationStorage.findAllByQuery({
+        exnSaid: previousExnGrantMsg.exn.d,
+      });
+
+      if (notifications.length) {
+        const notificationRecord = notifications[0];
+        if (
+          !notificationRecord.linkedGroupRequests ||
+          !Object.values(notificationRecord.linkedGroupRequests).includes(true)
+        ) {
+          notificationRecord.linkedGroupRequests = {
+            ...notificationRecord.linkedGroupRequests,
+            [exchange.exn.d]: false,
+          };
+        } else {
+          await Agent.agent.ipexCommunications.acceptAcdcFromMultisigExn(
+            exchange.exn.d
+          );
+          notificationRecord.linkedGroupRequests = {
+            ...notificationRecord.linkedGroupRequests,
+            [exchange.exn.d]: true,
+          };
+        }
+        await this.notificationStorage.update(notificationRecord);
+        return;
```

Comment:
If there's no notification, either:
- We've received this before the grant (unlikely) - I have a ticket to handle this. OR,
- The ACDC has appeared (threshold amount accepted it) and we're just receiving _another_ signing from say the THIRD wallet in a 2 of 3 multi-sig.

We need to return OUTSIDE this if statement to make sure it doesn't continue to go and create a `MultisigExn` notification. We never want to have `MultisigExn` for IPEX